toolbox/ansible_default_config.py

```diff
@@ -69,16 +72,22 @@ def _generate_config(component):
             ansible_arg_name = f"{component.ansible_role}_{arg}"
             print(yaml.dump({ansible_arg_name: default_value}).strip() if default_value != "" \
                   else f"{ansible_arg_name}:", file=f)
+            if i < len(mapped_params)-1:
+                print("", file=f)
+        
+        if len(ansible_constants) > 0:
             print("", file=f)
+            print("# Constants", file=f)
```

Comment:
why did you chose to add `# Parameters` & `# Constants`?
(I don't mind, don't remove it)

I'm not 100% sure myself why I chose to put in Python vs in Ansible, but I think it made more sense to have them side by side in the Python code, somehow like a C header files with prototypes and constants